content/tutorial/12-errors.md

```diff
@@ -49,7 +49,7 @@ You may have noticed the command doesn't do very much - in fact all it does is r
 - `CommandInvokeError` due to the content of the command raising `DivisionByZero`
 
 {{< tip "info" >}}
-To get the original error from a `CommandInvokeError`, you can use its `original` attribute, which returns the exception that cause it.
+To get the original error from a `CommandInvokeError`, you can use its `original` attribute, which returns the exception that causes it.
```

Comment:
This should actually have been `caused`, rather than `causes`

Reply:
Ah good point, I'll change that